src/recordlinker/schemas/algorithm.py

```diff
@@ -78,8 +77,24 @@ class Algorithm(pydantic.BaseModel):
     label: str = pydantic.Field(pattern=r"^[a-z0-9]+(?:-[a-z0-9]+)*$")
     description: typing.Optional[str] = None
     is_default: bool = False
+    include_multiple_matches: bool = True
+    belongingness_ratio: tuple[float, float]
```

Comment:
I recently [discovered a slightly better way to do this](https://github.com/CDCgov/RecordLinker/pull/126/files#diff-f8b75e7f2665e1eea4eee28b690b98fa9fd3b2687327a847c80df04145ecb069R40).  It won't remove the requirements for a separate validator function, as we still need to validate that the lower is less than or equal to the upper.  However, it will provide more documentation to the end user in the API on the valid range.

```python
belongingness_ratio: tuple[Annotated[float, Field(ge=0, le=1)], Annotated[float, Field(ge=0, le=1)]]
```